avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -198,11 +198,13 @@ cMaxnMass = 5
 interpOption = 2
 # minimum flow depth [m]
 hmin = 0.05
-# remesh the input DEM
+# remesh the input DEM or look for remeshed DEM 
 # expected mesh size [m]
 meshCellSize = 5
 # threshold under which no remeshing is done
 meshCellSizeThreshold = 0.001
+# force remesh - if True remeshing performed also if remeshed DEMs in Inputs/DEMremeshed available
+forceRemesh = True
```

Comment:
why is a forceRemesh necessary? Either the mesh is different from the specified resolution -> remesh, or it is the same -> do not remesh...